src/components/reduxStore/reducer/rootReducer.js

```diff
@@ -0,0 +1,6 @@
+import { combineReducers } from "redux";
+import loginReducer from "./loginReducer";
+
+export const rootReducer = combineReducers({
+    auth: loginReducer,
```

Comment:
Kindly consider changing the name of the reducer, to a more fitting name that is more generic to accommodate both login, logout, and sign up. consider a name like "authReducer, AuthenticationReducer".